props/common.props

```diff
@@ -6,6 +6,13 @@
     <NuGetAudit>true</NuGetAudit>
   </PropertyGroup>
 
+  <PropertyGroup Label="Package">
+    <Authors>Corniel Nobel, Wesley Baartman</Authors>
```

Comment:
`Authors` tag is delimited with semicolons (https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#pack-target).